### PR TITLE
Fixed warnings picked up by Analyzer

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**

### CODE REVIEW

File: APCAppCore/APCAppCore/Library/APCTaskImporter.h [hunk 1]
```diff
@@ -49,7 +49,7 @@
  
  See the .m file for technical notes on this method.
  */
-- (void) processTasks: (NSArray *) arrayOfTasks
+- (BOOL) processTasks: (NSArray *) arrayOfTasks
```
Comment (Erin-Mounts) [inline APCAppCore/APCAppCore/Library/APCTaskImporter.h:52]: What's the reason for changing this method's signature? Doesn't _seem_ like something the analyzer would advise, and I don't see any callers that have been updated in this PR.

File: APCAppCore/APCAppCore/Library/APCTaskImporter.m [hunk 2]
```diff
@@ -139,11 +139,12 @@ + (void) load
 #pragma mark - The Main Import Method
 // ---------------------------------------------------------
 
-- (void) processTasks: (NSArray *) arrayOfTasks
+- (BOOL) processTasks: (NSArray *) arrayOfTasks
```
Comment (Erin-Mounts) [inline APCAppCore/APCAppCore/Library/APCTaskImporter.m:142]: see previous question about method signature

File: APCAppCore/APCAppCore/Library/APCTaskImporter.h (hunk 1 above)
Comment (syoung-smallwisdom) [inline APCAppCore/APCAppCore/Library/APCTaskImporter.h:52]: If you are passing in an error pointer, they like for you to also return whether or not the method passed.

**Comment by Erin-Mounts** [comment]
looks good

**Pull request merged**